### ISSUE TITLE
Change script to load async

### ISSUE THREAD

**Issue by TalAter** [TalAter/UpUp#38]
Change the script to load asynchronously, as suggested by @slightlyoff.

**Comment by AkshatM** [comment]
What is wrong with the `async` attribute in HTML5 for this?

I understand Opera support is missing, but not sure what other shortcomings abound.

**Comment by TalAter** [comment]
I don't think that will be enough, as the other methods (e.g. `UpUp.start()`) won't work before the script loaded.
